tests/test_serialization.py

```diff
@@ -18,6 +18,7 @@
     UnsignedShort: [0, 400],
     UnsignedLong: [0, 400],
     Integer: [-1000, 1000],
+    FixedPointInteger: [float(-13098.3439), float(-0.83), float(1000)],
```

Comment:
The other test failure (for Python 3) is caused by the fact that `-13098.3439` can't be approximated as a `FixedPointInteger` to within 3 decimal places, as required by the test suite. The easiest way to deal with this is to just change it to one which can, such as `-13098.3435`.

Reply:
Whoops, it's rather `-0.83` which is offending value, not `-13098.3439`.

Reply:
I think making a case for `FixedPointInteger` that calls almostEquals with 2 digits might be better, I don't think 5 bits can represent a lot when converted to floats.

Reply:
Another way to deal with it would be to make a special case for `FixedPointInteger` in the code below, checking that the difference between the original and deserialised numbers is less than `1.0/32.0` (i.e. the resolution of a `FixedPointInteger`).

Reply:
Even 2 decimal digits would not be sufficient for all cases, such as this one:
```
>>> int(0.03124 * 32) / 32.0
0.0
```

Reply:
I'm afraid this is above my knowledge with python and its behaviour when dealing with rounding and division.
Any further help here appreciated. Thanks :D